packages/preferences/src/browser/views/preference-widget.tsx

```diff
@@ -50,6 +50,18 @@ export class PreferencesWidget extends Panel implements StatefulWidget {
         return this.tabBarWidget.currentScope;
     }
 
+    get availableTitles(): Title.Dataset[] {
+        return this.tabBarWidget.availableTitles;
+    }
+
+    openWithScope(selection: Preference.SelectedScopeDetails): void {
+        this.tabBarWidget.setNewScopeSelection(selection);
+    }
+
+    openWithQuery(query: string): void {
```

Comment:
minor: I think we should rename the method, as it does not actually `open`:

```suggestion
    updateSearchTerm(term: string): void {
```